x-pack/plugins/stack_connectors/common/sentinelone/constants.ts

```diff
@@ -10,12 +10,14 @@ export const SENTINELONE_CONNECTOR_ID = '.sentinelone';
 export const API_MAX_RESULTS = 1000;
 
 export enum SUB_ACTION {
-  KILL_PROCESS = 'killProcess',
```

Comment:
was this on purpose?

Reply:
yes. The implementation for kill-process will be done on our side and not the connector